src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
-spring.cloud.gcp.project-id=absolute-text-251105
-spring.cloud.gcp.credentials.location=file:/app/config/secret.json
+#spring.cloud.gcp.project-id=absolute-text-251105
+#spring.cloud.gcp.credentials.location=file:/app/config/secret.json
 #spring.cloud.gcp.credentials.location=file:/Users/ashishgupta/Downloads/secret.json
 #mail
 spring.mail.host=smtp.gmail.com
```

Comment:
uncomment the secret key location and project-id, also delete the commented lines.